src/middlewared/middlewared/plugins/chart_releases_linux/portals.py

```diff
@@ -71,8 +70,20 @@ def tag_func(key):
 
             cleaned_portals[portal_type] = t_portals
 
+        cleaned_portals.update(self.get_user_configured_portals(release_data, node_ip))
         return cleaned_portals
 
+    @private
+    def get_user_configured_portals(self, release_data, node_ip):
+        portals = {}
+        for portal_config in release_data['config'].get(CUSTOM_PORTALS_KEY) or []:
+            path = portal_config.get('path') or ''
+            host = node_ip if portal_config['useNodeIP'] else portal_config['host']
+            portals[portal_config['portalName']] = [
+                f'{portal_config["protocol"]}://{host}:{portal_config["protocol"]}{path}'
```

Comment:
```suggestion
                f'{portal_config["protocol"]}://{host}:{portal_config["port"]}{path}'
```
I think this needs to be port?